src/devices/Media/VideoDevice/Devices/UnixVideoDevice.cs

```diff
@@ -70,14 +96,21 @@ public override void Capture(string path)
         /// Capture a picture from the video device.
         /// </summary>
         /// <returns>Picture stream.</returns>
-        public override Stream Capture()
+        public override byte[] Capture()
```

Comment:
Why changing the Stream for a byte[]?

Reply:
If the consumer wants a `Stream`, they can easily make one themselves without incuring additional allocations. Where as, if the consumer wants a `byte[]` like I do, then you have to call ToArray() on the Stream incurring the additional allocation. While that is of course less important for the individual Capture call right now, I didn't want to prevent future changes where it may be more important, like allowing to Capture from multiple connections on a webserver.